### ISSUE TITLE
When sticky header enabled, there is a page jump when changing scroll direction on mobile

### ISSUE THREAD

**Issue by ConsultMePL** [elementor/elementor#20755]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

This is pretty much the same issue as #17767 which was marked as resolved - it took Elementor 4 months to fix it. It was fine for another 5 months until 3.8 when it broke again.

Basically when you have a sticky header enabled (not sure about other non-header options), then as soon as you stop scrolling AFTER you changed a scroll direction (so you were scrolling down and then changed direction going up), there is a page jump which is equal to the header's height.

Several people raised this issue in #17767 thread, obviously no one from Elementor bothered.

Why are you guys keep breaking things again and again? 

### Steps to reproduce

1. Enable sticky option for header
2.  Go to mobile version
3.  Scroll down and then scroll up, as soon as you stop, there will be a page jump equal to the height of header.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.11
	PHP Version: 7.4.33
	PHP Memory Limit: 1000M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 500M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://podjablonia.consultme.pl
	Home URL: https://podjablonia.consultme.pl
	WP Multisite: No
	Max Upload Size: 500 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: pl-PL
	Timezone: Europe/Warsaw
	Admin Email: marketing@consultme.pl
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: pl_PL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	Category Order and Taxonomy Terms Order
		Version: 1.7.4
		Author: Nsp-Code

	Custom background-image size for Elementor
		Version: 1.0.1
		Author: Michael Gangolf

	Elementor
		Version: 3.9.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.1
		Author: Elementor.com

	Hummingbird Pro
		Version: 3.4.0
		Author: WPMU DEV

	JetEngine
		Version: 3.1.0-beta1
		Author: Crocoblock

	JetFormBuilder
		Version: 2.1.11
		Author: Crocoblock

	JetTabs For Elementor
		Version: 2.1.20
		Author: Crocoblock

	Post Types Order
		Version: 1.9.9.2
		Author: Nsp Code

	Redis Object Cache
		Version: 2.2.2
		Author: Till Krüss

	Smush Pro
		Version: 3.12.3
		Author: WPMU DEV

	UiPress Lite
		Version: 3.0.94
		Author: Admin 2020

	UiPress Pro
		Version: 3.0.4
		Author: Admin 2020

	WPCodeBox
		Version: 1.4.1
		Author: WPCodeBox

	WPMU DEV Dashboard
		Version: 4.11.17
		Author: WPMU DEV

== Wykorzystanie elementów ==
	
	container : 7
		container : 4
		heading : 2
		jet-listing-grid : 4
	footer : 1
		container : 4
		heading : 3
		icon-list : 2
		theme-site-logo : 1
	header : 1
		container : 1
		icon : 1
		image : 1
		nav-menu : 1
	popup : 1
		container : 1
		form : 1
		heading : 1
	wp-page : 5
		button : 1
		container : 20
		form : 1
		heading : 12
		html : 1
		icon-box : 7
		icon-list : 1
		image : 2
		jet-listing-dynamic-field : 1
		jet-listing-grid : 5
		jet-tabs : 1
		text-editor : 3
	jet-listing-items : 0
		container : 16
		heading : 4
		icon : 2
		jet-listing-dynamic-field : 16
		jet-listing-dynamic-image : 3
		jet-listing-grid : 6
		text-editor : 1

== Eksperymenty z Elementorem ==
	Zoptymalizowany wynik DOM: Domyślnie aktywne
	Ulepszone ładowanie zasobów: Domyślnie aktywne
	Poprawione ładowanie CSS: Domyślnie aktywne
	Czcionka ikon: Włączono
	Ulepszenia w zakresie dostępności: Domyślnie aktywne
	Dodatkowe niestandardowe punkty zmiany: Domyślnie aktywne
	Import Export Website Kit: Domyślnie aktywne
	Ukryj natywne widgety WordPressa w wynikach wyszukiwania: Domyślnie aktywne
	admin_menu_rearrangement: Domyślnie nieaktywne
	Flexbox Container: Włączono
	Default to New Theme Builder: Domyślnie aktywne
	Hello Theme Header & Footer: Domyślnie aktywne
	Strony docelowe: Domyślnie aktywne
	Próbnik kolorów: Domyślnie aktywne
	Ulubione widgety: Domyślnie aktywne
	Górny pasek administratora: Domyślnie aktywne
	Lazy Load Background Images: Domyślnie nieaktywne
	Save as Default: Domyślnie aktywne
	Page Transitions: Domyślnie aktywne
	Notes: Nieaktywne
	Loop: Domyślnie aktywne
	Form Submissions: Domyślnie aktywne
	Scroll Snap: Domyślnie aktywne

== Dziennik ==
	
Log: showing 20 of 322022-11-22 16:34:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.8.2',
)]
2022-12-07 21:03:25 [info] elementor::elementor_updater Started 
2022-12-07 21:03:25 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-07 21:03:25 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-07 21:03:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 21:03:25 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 21:04:22 [info] elementor-pro::elementor_pro_updater Started 
2022-12-07 21:04:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-07 21:04:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-07 21:04:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 21:04:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-14 21:34:54 [info] elementor-pro::elementor_pro_updater Started 
2022-12-14 21:34:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-14 21:34:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-14 21:34:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.0',
  'to' => '3.9.1',
)]
2022-12-14 21:49:04 [info] elementor::elementor_updater Started 
2022-12-14 21:49:04 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-14 21:49:05 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-14 21:49:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0',
  'to' => '3.9.1',
)]
2022-12-14 21:49:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0',
  'to' => '3.9.1',
)]

JS: showing 5 of 5JS: 2022-11-22 13:49:33 [error X 1][https://podjablonia.consultme.pl/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:649119] this.model.isValidChild is not a function 
JS: 2022-11-22 23:46:29 [error X 15][https://podjablonia.consultme.pl/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:769459] Cannot read properties of undefined (reading \'localized\') 
JS: 2022-11-24 12:49:23 [error X 6][https://podjablonia.consultme.pl/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.8.2:3:99294] Cannot convert undefined or null to object 
JS: 2022-11-24 15:07:36 [error X 15][https://podjablonia.consultme.pl/wp-content/plugins/jet-tabs/assets/js/jet-tabs-frontend.min.js?ver=2.1.20:1:5806] Failed to execute \'observe\' on \'MutationObserver\': parameter 1 is not of type \'Node\'. 
JS: 2022-12-14 20:49:34 [error X 5][https://podjablonia.consultme.pl/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading \'replace\') 

PHP: showing 5 of 5PHP: 2022-11-23 22:27:47 [error X 1][/home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/core/documents-manager.php::493] Uncaught Error: Call to a member function is_built_with_elementor() on bool in /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/core/documents-manager.php:493
Stack trace:
#0 [internal function]: Elementor\Core\Documents_Manager->ajax_save(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /home/consultme/domains/consultme.pl/public_htm [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-24 02:42:10 [notice X 6][/home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/includes/base/controls-stack.php::1430] Undefined index: _background_hover_image_tablet [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-24 02:55:12 [notice X 5][/home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/includes/base/controls-stack.php::1430] Undefined index: image_height_tablet [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-24 10:33:30 [error X 1][/home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::294] Uncaught Error: Call to a member function update_meta() on null in /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:294
Stack trace:
#0 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(6, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/consultme/domains/consultme.pl/public_html/podjablonia/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-24 15:58:03 [notice X 67][/home/consultme/domains/consultme.pl/public_html/podjablonia/wp-content/plugins/elementor/includes/base/controls-stack.php::1430] Undefined index: field_bg_image_tablet [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Custom background-image size for Elementor: Kompatybilność nieokreślona
	Elementor Pro: Kompatybilność nieokreślona
	JetTabs For Elementor: Kompatybilność nieokreślona

== Elementor Pro - Compatibility Tag ==

**Comment by ConsultMePL** [comment]
Ok, after a lot of digging, I found what was causing the issue.

In order to set a proper 100vh height on mobile (the issue of the url bars) I've used a custom js along with this css code:
height: calc(var(--vh, 1vh) * 100)!important;

however, recently dvh started being supported as well.

Whichever method I use along with the containers (custom code or dvh), there is this strange screen jump.

But as mentioned before - it's been fixed for a while.

@nicholaszein anyone can have a look at this? From the UI perspective it's pretty bad... and what's annyoing is that it was fixed.

So to sum up in order for the issue to exist, there has to be a custom height for VH so it's a reall 100vh on mobile, containers have to be enabled as well as sticky header.

Since you guys fixed it in the mast, just have a look what you have done...

**Comment by nicholaszein** [comment]
Hello @ConsultMePL,

thanks for reporting this, and apologies for the delay! 🙏 

I can reproduce the issue, but not in a consistent way. It only happens in specific cases when you set `100vh` to any element on the page. This happens due to how mobile browsers handle the `position` property, and it's indeed very tricky. I will check with the team to see what can be done to fix this again. Let's see if we can find a way to set this right.

Thank you for your efforts!

Kind regards

**Comment by theitpyme** [comment]
Hello Everyone, any news regarding this Issue? I have the same issue in one of my websites and I would like to solve it :)

Thanks in advance,

Kind Regards.

> Hello @ConsultMePL,
> 
> thanks for reporting this, and apologies for the delay! 🙏
> 
> I can reproduce the issue, but not in a consistent way. It only happens in specific cases when you set `100vh` to any element on the page. This happens due to how mobile browsers handle the `position` property, and it's indeed very tricky. I will check with the team to see what can be done to fix this again. Let's see if we can find a way to set this right.
> 
> Thank you for your efforts!
> 
> Kind regards

**Comment by BriceGdT** [comment]
Hello there,
same exact issue too, is there a workaround?
Thanks a lot,
B.

**Comment by webcoreinc** [comment]
Jet pligins are buggy as hell. The reason we canceled our subscription.  Their devs are very sub par.  Seems like they're using Fiverr devs or something.
This issue still exists and likely will bever be fixed knowing elementor.
